### ISSUE TITLE
kamailio 5.0 for xenial

### ISSUE THREAD

**Issue by tverlaan** [sipwise/kamailio-deb-jenkins#7]
Hi,

I don't see kamailio 5.0/dev for Xenial. On the nightly binaries page for Xenial it has kamailio 4.4.3. I know I recently added it, did I forget to add something?

Thanks for all the nice work!

**Comment by linuxmaniac** [comment]
They do exist already. Maybe you are missing them in the list at http://deb.kamailio.org/index.html
dev: http://deb.kamailio.org/kamailiodev-nightly/dists/xenial/
4.4: http://deb.kamailio.org/kamailio44-nightly/dists/xenial/

**Comment by linuxmaniac** [comment]
If you are missing 4.4.3 at that time the xenial support was not ready. It will be there for next 4.4.4

**Comment by tverlaan** [comment]
Yeah, I actually ment 4.4.4. But thanks for adding it to the list!

**Comment by tverlaan** [comment]
I think I found out why I couldn't find kamailio 5.0 for xenial. The packaged version doesn't contain the actual kamailio version. See:

```
$ dpkg -l | grep kamailio
ii  kamailio                           4.4.3+0~20161102005433.637+xenial amd64        very fast and configurable SIP proxy

$ kamailio -V
version: kamailio 5.0.0-dev7 (x86_64/linux)
```

**Comment by linuxmaniac** [comment]
that should be addressed by kamailio/kamailio@11de68777142a82d093e1095ff75243e7cfa387a so next build should be OK
